aten/src/ATen/native/quantized/cpu/qlinear_prepack.cpp

```diff
@@ -207,6 +208,50 @@ c10::intrusive_ptr<LinearPackedParamsBase> PackedLinearWeightFp16::prepack(
 #endif // USE_FBGEMM
 
 #if AT_MKLDNN_ENABLED()
+inline ideep::tensor pack_weight(
+    const at::Tensor& weight,
+    torch::List<int64_t>& input_shape) {
+  std::vector<int64_t> w_dims = weight.sizes().vec();
+  ideep::tensor wei = ideep::tensor({w_dims, dnnl::memory::data_type::s8}, weight.data_ptr());
+  wei.transpose_(0, 1); // ONEDNN requires transposed weight
+  ideep::dims input_dims = input_shape.vec();
+  auto w_desc = ideep::matmul_forward::expected_weights_desc(wei.get_dims(), dnnl::memory::data_type::s8,
+                                                             dnnl::memory::data_type::u8);
+  wei = wei.reorder_if_differ_in(w_desc);
+  return wei;
+}
+
+inline ideep::tensor pack_bias(
+    const at::Tensor& b,
+    int64_t oc,
+    ideep::scale_t weight_scales = ideep::scale_t(),
+    double input_scale = 1,
+    int64_t input_zero_point = 0) {
```

Comment:
Let's design for onednn 3.x in which bias packing is not needed. Then, the code here can be simplified further.